commit 7ad56e8480e6f12f1a4c128774020400da87c86d (refs/changes/76/1576/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-06-20T08:25:57-07:00 (1 year, 4 months ago)
    
    OS-5985 vmadm man page incorrectly states dns_domain property is associated with /etc/hosts

diff --git a/src/vm/man/vmadm.1m.md b/src/vm/man/vmadm.1m.md
index cd419449..fdd4e36a 100644
--- a/src/vm/man/vmadm.1m.md
+++ b/src/vm/man/vmadm.1m.md
@@ -954,6 +954,9 @@ tab-complete UUIDs rather than having to type them out for every command.
 
     dns_domain:
 
+        For OS VMs this specifies the search value for /etc/resolv.conf that
+        gets set at create time. Updating this after create will have no
+        effect unless maintain_resolvers is true.
         For OS VMs this specifies the domain value for /etc/hosts that gets set
         at create time. Updating this after create will have no effect.
 
